### ISSUE TITLE
Investigate MorphDOM vs ReactDOM

### ISSUE THREAD

**Issue by Archmonger** [idom-team/idom#883]
### Discussed in https://github.com/idom-team/idom/discussions/798

<div type='discussions-op-text'>

<sup>Originally posted by **Archmonger** August 10, 2022</sup>
Currently, due to the addition of our script tag, there is the potential of VDOM/DOM mismatches. This can cause client side exceptions within the ReactDOM library.

This discussion exists to debate whether MorphDOM should be considered as an alternative, or whether we should simply work around these issues with ReactDOM.</div>

TLDR: Script tag can be used to modify the DOM directly. We may want to consider MorphDOM to account for this.
